File: C#/projects/ProductsNCatogories/Models/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ProductsNCatogories.Models
{
    public class Product
    {
        [Key]
        public int ProductId{get;set;}
        [Required(ErrorMessage="is Required")]
        [MinLength(2,ErrorMessage="Must be at least 2 characters")]
        public string Name{get;set;}

        [Required(ErrorMessage="is Required")]
        [MinLength(2,ErrorMessage="Must be at least 2 characters")]
        public string Description{get;set;}

        [Required(ErrorMessage="is Required")]
        public double Price{get;set;}
        public DateTime CreatedAt { get; set; } = DateTime.Now;
        public DateTime UpdatedAt { get; set; } = DateTime.Now;

        public List<ProductsInCaegory> ProductsInCaegories{get;set;}

    }
}

File: MERN/Projects/Product-Project/server/models/Product.js
const mongoose=require('mongoose');

const ProductSchema=new mongoose.Schema({
    title:{
        type:String,
        required: true,
        minlength:[2,'Product must be 2 or more characters!']
    },
    price:{
        type:Number,
        required:true
    },
    description:{
        type:String,
        required:true
    }

},{timestamps:true})

mongoose.model("Product",ProductSchema);